### PR TITLE
Update CHANNELS.md

### PR DESCRIPTION
**Pull request by v-kydela**

### CODE REVIEW

File: content/CHANNELS.md [hunk 1]
```diff
@@ -8,20 +8,21 @@
 
 ## TL;DR
 1. Update your Emulator to be on version `4.4`. You can do this by selecting `Help` -> `Check for Update...`
-1. Install [ngrok](https://ngrok.com/) and navigate to the ngrok executable's location in a terminal.
-2. Run ngrok `./ngrok http 3979 --host-header=localhost`
-3. Open your bot in the [Azure Portal](https://ms.portal.azure.com/), select the `Settings` blade and paste the ngrok url provided in the terminal into the *Messaging Endpoint* field then save. **note:** If necessary, don't forget to add the `/api/messages` endpoint.
-4. Set the appropriate environment variables (prefix with EXPORT for OSX/Linux, SET for Windows):
+2. Install [ngrok](https://ngrok.com/) and navigate to the ngrok executable's location in a terminal.
```
Comment (corinagum) [inline content/CHANNELS.md]: These numbers can all be changed to 1 and markdown will number them appropriately. That way if more steps are added the numbers don't have to be fixed manually.

Comment (v-kydela) [reply content/CHANNELS.md]: Okay done
